tests/golden/vault/common.bats

```diff
@@ -0,0 +1,35 @@
+# SPDX-FileCopyrightText: 2020 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+#!/usr/bin/env bats
+
+load '../helpers/bats-support/load'
+load '../helpers/bats-assert/load'
+load '../helpers'
+
+@test "vault bad path" {
+  run coffer view "$(echo -e "bad\npath")"
+
+  assert_failure
+  assert_output --partial - <<EOF
+Internal backend error:
+Backend returned a path segment that is not a valid entry or directory name.
```

Comment:
> Backend returned a path segment that is not a valid entry or directory name

The message seems off. This path was returned by the backend; it was entered by the user.

Reply:
done